app/scripts/controllers/eventDetails.js

```diff
@@ -25,9 +25,21 @@ angular.module('confRegistrationWebApp')
     };
     $scope.changeTab($scope.tabs[0]);
 
-    $scope.paymentGateways = [
-      {id: 'AUTHORIZE_NET', name: 'Authorize.Net'}
-    ];
+    $scope.paymentGateways = {
+      AUTHORIZE_NET: {
+        name: 'Authorize.Net',
+        fields: {
+          paymentGatewayId: { title: 'Account ID' },
+          paymentGatewayKey: { title: 'Key' }
+        }
+      },
+      TSYS: {
+        name: 'TSYS',
+        fields: {
+          paymentGatewayId: { title: 'Merchant Device' }
```

Comment:
We actually should probably not name it `merchant` since there exists a `merchant id` as distinct from a `device id`. I had thought `merchant` was more UI friendly, but it's actually not accurate and could be confusing as well as potentially wrong if an actual `merchant id` is provided. 

For now, let's rename this `ERT Device ID`.

@canac